test/parse/TestPythonParser.cpp

```diff
@@ -0,0 +1,101 @@
+#include <array>
+#include <boost/test/tools/old/interface.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "parse/Parse.h"
+#include "parse/PythonParser.h"
+#include "universe/Effect.h"
+#include "universe/Tech.h"
+#include "universe/UnlockableItem.h"
+#include "util/Directories.h"
+#include "util/GameRules.h"
+#include "util/Pending.h"
+
+#include "ParserAppFixture.h"
+
+BOOST_FIXTURE_TEST_SUITE(TestPythonParser, ParserAppFixture)
+
+BOOST_AUTO_TEST_CASE(parse_game_rules) {
+    PythonParser parser(m_python);
+
+    Pending::Pending<GameRules> game_rules_p = Pending::Parsed(parse::game_rules, parser, m_scrpiting_dir / "game_rules.focs.py");
```

Comment:
Actually, I'd like to have two test sets, grouped by change reason:

- First will parse the real game files and assert nothing.  This test set will check if a real game file is syntactically correct.
  You don't want to change the test just because someone adds a new rule or rename it.
- The second set will have its own test files (I'd recommend having different files for the different tests).  You can add any case you want, you can put self-explanatory values inside, you can have erroneous testcases. And the only reason to change these files is the parser update.

Reply:
We have first test set which parse real game files in system tests where the client and the server call all parsers.

Second sounds reasonable, so we need to have to copy some subset of default to test and use it for tests.

Reply:
I'd prefer not copying, but creating from scratch.  There is no reason to make it look like a game object, it should represent a minimum field required for tests.  It is also a good practice to have an independent fixture for each tests, but for the first implementation, it is not an option.

Reply:
I've tried it to do but got issue where to place test resources. Tests binaries are generated in `CMAKE_BINARY_DIR` and `default` folder is linked here from `CMAKE_SOURCE_DIR` but I not sure if it good idea to place it here.
Maybe I should create `default-test` folder and then link it before call `unittest` target or after test binaries built.